CHANGES.rst

```diff
@@ -1,6 +1,13 @@
 Changes
 =======
 
+Version 0.7.1 (2020-11-09)
+--------------------------
+
+- Adds support for ``htcondor_max_runtime`` and ``htcondor_accounting_group`` in HTC jobs.
+- Adds support for restarting yadage workflows (through ``accept_metadir`` operational option).
+- Adds REANA server version to ``reana-client ping`` output.
```

Comment:
WRT first point: HTCondor instead of HTC, see the usual phrasing:

- Adds support for specifying ``htcondor_max_runtime`` and ``htcondor_accounting_group`` for HTCondor compute backend jobs.

WRT second point, Yadage with capital Y.  But shouldn't we say CWL and Yadage, i.e. both?  Also, I would say `Fixes restarting..." because this is more of a bug fix, which is why we put it to 0.7.1. So:

- Fixes restarting of CWL and Yadage workflows.

WRT third point, perhaps we should say: "Changes..." following the same feature/improvement/bugfix logic, so:

- Changes `ping` command output to include REANA client and server version information.

However, the first two points are not really thanks to the changes in the `r-client` itself, more thanks to the changes in the cluster. So we should probably leave them out from here and put them into the main repository `reana`'s release notes. Which would leave us with the only point about ``ping`` command.